sylt-std/src/sylt.rs

```diff
@@ -379,8 +379,8 @@ sylt_macro::extern_function!(
     "sylt_std::sylt"
     as_str
     "Converts to a string representation"
-    [One(Int(i))] -> Type::String => {
-        Ok(Value::String(Rc::new(i.to_string())))
+    [One(v)] -> Type::String => {
+        Ok(Value::String(Rc::new(v.to_string())))
```

Comment:
:+1:

Reply:
This is what is hindering tuples to be passed in. Let me know if you want help with writing this without the macro - can be hard to do if you haven't done it before.